### ISSUE TITLE
Chickens can bust out through their containment in hilariously dumb fashion

### ISSUE THREAD

**Issue by ktccd0** [VOREStation/VOREStation#12115]
#### Brief description of the bug
Chickens are able to break out of the little farm area in botany.

#### What you expected to happen
Chickens shouldn't be able to crush the walls or doors to escape.

#### What actually happened
A player can control a chicken and just bash their way out through the windows.

#### Steps to reproduce
- (Step 1) Become chicken
- (Step 2) Bash windows
- (Step 3) Freedom!

#### Anything else you may wish to add:
- (Location if it's a mapping issue, screenshots, sprites, etc.)
It's on the NRV Stellar Delight map

**Comment by Runa-Dacino** [comment]
> I don't realy see the issue here since this is player-controlled behavior instead of AI behavior.

Simple-mobs indeed cannot figure out how to climb through the low wall. (leading to them instead spending 15 minutes ramming the low wall instead).

**Comment by ktccd0** [comment]
Eh, I figured I'd make an issue on the hub here so I could know. I could see arguments for either case, honestly.
Like, yeah, of course, a player behaviour is different from a mob, I'm not here to complain about someone's behavior, I'm entirely here because of programming reasons.

I was curious if this was intended on a mechanical level, for tiny mobs to be able to crush reinforced glass, for the same reasons that I vaguely recall that a player-mob can't punch through them with their fists?


So, basically, I was intending to bring this to attention on the off chance that some part of the code accidentally gave all player-controlled mobs too much strength or something. These strange situations can often be the result of weird code interactions and it's worth checking that it's intentional. (Well, IMO it's worth checking, but that's an opinion and I am not an authority on the subject)

**Comment by Heroman3003** [comment]
Intentional behaviour in term of chicken strength, not intentional in term of player action.
